src/test/scala/nl/knaw/dans/easy/pid/generator/DatabaseSpec.scala

```diff
@@ -83,9 +84,13 @@ class DatabaseSpec extends TestSupportFixture with SeedDatabaseFixture with Data
   it should "return the related timestamp if the requested PID is already in the database" in {
     val pid = "testpid"
     val created = new DateTime(1992, 7, 30, 16, 2, timeZone)
+    val createdFormatted = created.toString(ISODateTimeFormat.dateTimeNoMillis())
+      .replace("T", " ")
+      .replace("Z", "+0:00")
```

Comment:
what change do you actually propose? HSQLDB requires this date/time as `2018-07-30 16:01:02+0:00` with a **space** instead of `T` and `+0:00` instead of `Z`. If timezone is not equal to UTC, it already has a `+/-h:mm` postfix.

Reply:
I wasn't aware about your last phrase.